frontend/src/model/photo.js

```diff
@@ -447,6 +449,11 @@ export class Photo extends RestModel {
 
   videoUrl() {
     let file = this.videoFile();
+    const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
```

Comment:
Just to be sure and save some time for reverse engineering: What exactly does this regex do, especially with respect to Chrome and Android?

In our existing Safari browser check, we simply use `navigator.userAgent.indexOf("Safari/")`, which I might be too simplistic?